pkg/report/table/table.go

```diff
@@ -52,26 +53,33 @@ type Renderer interface {
 
 // Write writes the result on standard output
 func (tw Writer) Write(report types.Report) error {
+	pkgs := map[string][]ftypes.Package{}
+	for _, result := range report.Results {
+		if result.Class == types.ClassOSPkg || result.Class == types.ClassLangPkg {
+			pkgs[result.Target] = result.Packages
+		}
+	}
+
 	for _, result := range report.Results {
 		// Not display a table of custom resources
 		if result.Class == types.ClassCustom {
 			continue
 		}
-		tw.write(result)
+		tw.write(result, pkgs)
 	}
```

Comment:
2 identical loops are performed here.
perhaps we can use 1 loop and store `pkgs` inside loop. We can also use slice(`pkgs :=[]ftypes.Package{}`)
What do you think?